developers/testing.md

```diff
@@ -188,11 +188,11 @@ One thing to note is that while the normal rake test command will migrate the da
 not (in the interest of saving time). To do this, simply run this in your foreman directory:
 
 ```
-RAILS_ENV=test rake db:drop && RAILS_ENV=test rake db:create && RAILS_ENV=test rake db:migrate
+RAILS_ENV=test rake db:drop && RAILS_ENV=test rake db:create && RAILS_ENV=test rake db:test:prepare && RAILS_ENV=test rake db:migrate
 ```
```

Comment:
@jlsherrill it looks like it only checks for pending migrations and will reload the schema if there is none there, but otherwise warn that there are migrations pending, so I am guessing a migration would still be necessary if there are migrations needed? 
http://stackoverflow.com/questions/15169894/what-does-rake-dbtestprepare-actually-do